views/discover/index.js

```diff
@@ -1,12 +1,56 @@
 var view = require('../../lib/view');
-var fakeDiscovery = require('../../lib/fake-discovery');
+var Firebase = require('firebase');
+var config = require('../../config/index.js');
+
+var fb = new Firebase(config.FIREBASE_URL);
 
 module.exports = view.extend({
     id: 'discover',
     template: require('./index.html'),
+    ready: function () {
+        this.$on('switchValueChanged', function (event) {
+            if (event === 'Newest') {
+                this.showNewest();
+            } else if (event === 'Featured') {
+                this.showFeatured();
+            }
+        });
+
+        this.showFeatured();
+    },
+    methods: {
+        showNewest: function () {
+            var self = this;
+
+            self.mode = 'Newest';
+
+            fb
```

Comment:
I'm making a new ticket for this since it's more complex than it should be. :imp: